media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
`.search-column` changes from `460px` width at desktop break-point to `340px` width at tablet break-point. The language-search form has a total maximim width of `710px` here, so there should be plenty of room to avoid having to change the width to such a small value here.